src/AbstractServer.php

```diff
@@ -72,20 +69,31 @@ protected function _buildCallback(Reflection\AbstractFunction $reflection)
         return $callback;
     }
 
+    /**
+     * Build callback for method signature
+     *
+     * @deprecated Since 2.7.0; method will be removed in 3.0, use
+     *             buildCallback() instead.
+     * @param  Reflection\AbstractFunction $reflection
+     * @return Method\Callback
+     */
+    // @codingStandardsIgnoreStart
+    protected function _buildCallback(Reflection\AbstractFunction $reflection)
+    {
+    // @codingStandardsIgnoreEnd
```

Comment:
Now, as we have the other method, I believe we can trigger user deprecation here (unless we do not want to do it for some reason?)